examples/bri-3/components/identity/bpiSubject/bpiSubject.interface.ts

```diff
@@ -0,0 +1,9 @@
+import { BpiSubjectType } from "./bpiSubjectType.enum";
+
+export interface IBpiSubject {
+    id: string; // TODO: Add uuid after #491
+    name: string;
+    description: string;
+    type: BpiSubjectType;
+    publicKey: string;
```

Comment:
@ognjenkurtic @Kasshern @Manik-Jain @kthomas -> thoughts on using an array here in order to be able to accommodate more keys and key types? See below.

```
keys: publickeys[];
publickeys: {
                    type: object,
                    properties: {
                            publickey: string,
                            keyType: string
                       }
                      }
```

Reply:
Definitely but it will come later as we start to deal with keys, signatures and related functionalities defined in the standard. Now it is more of a placeholder.